### PR TITLE
Add support for user-provided bookmarks.

### PR DESCRIPTION
**Pull request by j-jorge**
Please review this PR which adds support for a new content type in the form of bookmarks provided by the users. A bookmark is a link to a website along with a short description. Like other contents on the site this one can be commented, rated, tagged.

This commit provides the basics of the features. Registered users can submit and comment the bookmarks, anyone can read them.

Missing features include at least an atom feed and the forms for moderation.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -18,16 +18,23 @@
         post :move
       end
     end
+    resources :liens, controller: "bookmarks", as: "bookmarks", except: [:index, :new, :create] do
+      member do
+        post :convert
```
Comment (nono) [inline config/routes.rb]: I think you don't need this route

File: db/schema.rb [hunk 2]
```diff
@@ -106,6 +106,19 @@
...
+    t.string   "cached_slug",       limit: 165
+    t.integer  "owner_id",          limit: 4
+    t.string   "link",              limit: 1024,        null: false
+    t.string   "lang",              limit: 2, null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  add_index "bookmarks", ["cached_slug"], name: "index_bookmarks_on_cached_slug", using: :btree
+  add_index "bookmarks", ["owner_id"], name: "index_bookmarks_on_owner_id", using: :btree
```
Comment (nono) [inline db/schema.rb]: I don't see the migration for this. Did you forget to include it in the commits?

File: config/routes.rb (hunk 1 above)
Comment (j-jorge) [reply config/routes.rb]: Fixed.

File: db/schema.rb (hunk 2 above)
Comment (j-jorge) [reply db/schema.rb]: I did not write it. Now it's done :)

**Pull request merged**